graphql-schema.json

```diff
@@ -498,6 +554,22 @@
             },
             "isDeprecated": false,
             "deprecationReason": null
+          },
+          {
+            "name": "campaignCount",
```

Comment:
What's the benefit of this over `campaigns.length`?

Reply:
The client does not need to request all the campaign data to see the campaign count 😃

Reply:
I keep forgetting how good separate resolvers are D: